src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,35 @@
+.stars {
+  height: 16px;
+  display: flex;
+  flex-direction: row;
+
+  &__star--active {
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star-active.svg);
+    box-sizing: border-box;
+    align-items: center;
+    margin: 0 4px 0 0;
+    background-position: center;
+    background-repeat: no-repeat;
+  }
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star.svg);
+    box-sizing: border-box;
+    align-items: center;
+    margin: 0 4px 0 0;
+    background-position: center;
+    background-repeat: no-repeat;
+  }
+
+  &__star--active:last-child {
+    margin-right: 0;
+  }
+
+  &__star:last-child {
+    margin-right: 0;
```

Comment:
The `:last-child` pseudo-class is used to target the last element in a container. However, it seems unnecessary here as the `.stars__star` and `.stars__star--active` are likely to be used within the same container. If that's the case, you can combine these selectors to avoid redundancy.